pkg/services/k8s/apiserver/restoptionsgetter.go

```diff
@@ -46,7 +47,7 @@ func ProvideRESTOptionsGetter(cfg *setting.Cfg, features featuremgmt.FeatureTogg
 }
 
 func (f *RESTOptionsGetter) GetRESTOptions(resource schema.GroupResource) (generic.RESTOptions, error) {
-	if resource.Resource == "grafanaresourcedefinitions" {
+	if resource.Resource == "grafanakinds" {
 		return f.fallback.GetRESTOptions(resource)
```

Comment:
@ryantxu is there anything preventing us from storing kinds in entity api at this point?